keystorage/aws/src/main/java/tech/pegasys/signers/aws/SecretValueConverterUtil.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2023 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.signers.aws;
+
+import java.util.Objects;
+import java.util.Set;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.function.BiFunction;
+import java.util.stream.Collectors;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class SecretValueConverterUtil {
```

Comment:
nit: naming - probably don't need Util. 
'Convertor' in class, 'map' in method - IMO either SecretValueMapper or convertSecretValue

Reply:
> probably don't need Util.

On second thoughts, I guess this is a Util for the mapper function itself, so maybe makes sense

Reply:
Is it more of a SecretValueExtractor? Maybe could separate the concerns of extracting and then do the mapping afterwards, or would that be poor performance?